src/main/java/Main.java

```diff
@@ -0,0 +1,23 @@
+import java.io.IOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Arrays;
+import java.util.Scanner;
+
+public class Main {
+    public static void main(String[] args) throws ConflictException, IndefiniteException {
```

Comment:
This function should not throws exception, them will be catched inside and error message should be displayed